validation/standard-config-mainnet.toml

```diff
@@ -0,0 +1,23 @@
+# Standard Chain Config Parameters
```

Comment:
once https://github.com/ethereum-optimism/specs/pull/134 is merged, we should revisit this file to ensure everything matches

Reply:
https://github.com/ethereum-optimism/superchain-registry/issues/194